Packs/McAfee_ESM_v10/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
@amshamah419 / @yuvalbenshalom validate fails on the rn.
reason: yana changed the file path(not pack name) as it was needed for the UT.
if it passes 6.0 and updates correctly - are we ok with it? or do you think an issue could rise?